### DISCUSSION TITLE
Can‘t update the integration

### DISCUSSION THREAD

**Discussion by kdkmd57** [SukramJ/aiohomematic#1597] [Q&A]
### The problem

I can‘t update the integration, due to the reason that the integration os not listed in HACS . Actually is running version 1.33.1 but I don’t see any update or re-download button anywhere?
Ist it possible to install the actual versions manually?

### What version of HomematicIP (local) has the issue?

1.33.1

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

RaspberryMatic Standalone

### Which version of your homematic backend are you running?

3.75

### What hardware are you running for your system?

Procxmox

### Which config details do you use

- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [X] Homematic (Bidcos-RF)
- [X] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

**Comment by SukramJ** [comment]
Basically installation and updating is not part of this integration, and normally done by HACS.
So any problem with installation and updating should be posted [there](https://github.com/hacs/integration/issues).
If you manually copy the [homematicip_local](https://github.com/danielperna84/custom_homematic/tree/devel/custom_components/homematicip_local) directory into your custom_components folder, then this specific version will run, but there are no automatic updates.
I personally don't support these scenarios, so you better solve your HACS related problems.

**Reply by kdkmd57** [reply]
Thanks for your answer … I have tried to install a new homematic(ip) local integration via HACS on a new/ blank HA installation - but also there I cannot found the homemtic(ip) local integration in HACS.

**Reply by SukramJ** [reply]
Which installation instruction do you use?
Did you add the repo?

**Reply by kdkmd57** [reply]
I used the „official“ instruction (hacs.xyz/docs/setup/download) and downloaded with the „wget“ command and configured as it is shown at these website.
Afterwards I search for the integration in Hacs & the HA Integration side.
Have I install an additional repository between the steps?

**Reply by SukramJ** [reply]
Which installation instruction do you use for THIS integration?

**Comment by kdkmd57** [comment]
I forgot the repository - I did not see that the entry „Installation“ is a link, I that that this is an index an the chapter is mentioned below in the text.
Sorry & thank you so much for the hint with the repository
